ci-operator/config/openshift/loki/openshift-loki-release-6.0.yaml

```diff
@@ -0,0 +1,161 @@
+base_images:
+  base:
+    name: "4.16"
+    namespace: ocp
+    tag: base-rhel9
+  loki-tag:
+    name: v3.1.0
+    namespace: logging
+    tag: loki
+  lokistack-gateway:
+    name: mirror-5.y
+    namespace: logging
+    tag: lokistack-gateway
+  ocp_builder_rhel-9-golang-1.21-builder-multi-openshift-4.16:
+    name: builder
+    namespace: ocp
+    tag: rhel-9-golang-1.21-builder-multi-openshift-4.16
+  opa-openshift:
+    name: mirror-5.y
+    namespace: logging
+    tag: opa-openshift
+build_root:
+  image_stream_tag:
+    name: builder
+    namespace: ocp
+    tag: rhel-9-golang-1.21-builder-multi-openshift-4.16
+images:
+- dockerfile_path: Dockerfile.ocp
+  from: base
+  inputs:
+    ocp_builder_rhel-9-golang-1.21-builder-multi-openshift-4.16:
+      as:
+      - registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.21-openshift-4.16
+  multi_arch: true
+  to: loki
+- dockerfile_path: Dockerfile.promtail.ocp
+  from: base
+  inputs:
+    ocp_builder_rhel-9-golang-1.21-builder-multi-openshift-4.16:
+      as:
+      - registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.21-openshift-4.16
+  to: promtail
+- context_dir: operator
+  dockerfile_path: Dockerfile
+  from: base
+  multi_arch: true
+  to: loki-operator
+operator:
+  bundles:
+  - as: loki-operator-bundle
+    context_dir: operator/bundle/openshift
+    dockerfile_path: bundle.Dockerfile
+  substitutions:
+  - pullspec: quay.io/openshift-logging/loki-operator:0.1.0
+    with: pipeline:loki-operator
+  - pullspec: quay.io/openshift-logging/loki:v3.1.0
+    with: pipeline:loki-tag
```

Comment:
Similar to the beginning: I think this can be `loki` without the images in the build.